src/Parser/GeoJsonParser.js

```diff
@@ -212,6 +211,15 @@ export default {
         options.withNormal = options.withNormal == undefined ? true : options.withNormal;
         options.withAltitude = options.withAltitude == undefined ? true : options.withAltitude;
 
+        let filteringExtent;
+        if (options.filteringExtent) {
+            if (typeof options.filteringExtent == 'boolean') {
```

Comment:
what happens if filteringExtent is False?

Reply:
`options.filteringExtent` is checked line 215 then `filteringExtent` is undefined and there is not filtering.